### ISSUE TITLE
[keras]The model size is 250MB, but it takes up 800MB of memory after loading

### ISSUE THREAD

**Issue by DaPenggg** [tensorflow/tensorflow#21330]
The keras model(h5) size is 250MB, but it takes up 800MB of memory after loading

does anyone can explain the reason?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 30 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by DaPenggg** [comment]
bytes problem. I got it.
